react/components/SearchBar/index.js

```diff
@@ -53,8 +53,12 @@ function SearchBarContainer(props) {
     setInputValue(event.target.value)
   }, [])
 
+  const validateInputValue = useCallback(input => {
+    return input.replace(/\//g, '-')
+  }, [])
+
   const handleGoToSearchPage = useCallback(() => {
-    const search = inputValue
+    const search = validateInputValue(inputValue)
```

Comment:
I think a `sanitize` or `format` would better fit this method's usage

```suggestion
    const search = sanitizeInputValue(inputValue)
```